packages/react-server-website/deployment/deploy.yml

```diff
@@ -11,10 +11,18 @@
     - name: Copy NGINX configuration
       copy: src=../nginx.conf dest=/home/ubuntu/nginx.conf mode=0644
 
-    - name: Pull containers
-      shell: docker-compose pull && docker-compose build
+    - name: Pull images
+      command: docker-compose pull
```

Comment:
previously, we docker-compose pulled AND docker-compose built, but now it seems like we don't build anymore?  Is that expected?